src/schema/v2/artwork/index.ts

```diff
@@ -892,9 +892,15 @@ export const ArtworkType = new GraphQLObjectType<any, ResolverContext>({
         type: GraphQLBoolean,
         description: "Is this work only available for shipping domestically?",
         resolve: (artwork) => {
+          const domesticShippingSupported =
+            artwork.domestic_shipping_fee_cents ||
```

Comment:
Would the free domestic shipping (0) return true here as we want? I was not 100% sure if we need it to be `!=` null as well.

Reply:
excellent catch, adjusted

Reply:
Will always forget this 😭 

0 truthy in ruby
```ruby
irb(main):002:0> !!0
=> true
```

0 falsy in JS
```js
> !!0
=> false
```

Reply:
Impossible to remember as it has 0(true/false) logical sense! Basically random!